modules/ROOT/partials/misc/inline-formatting-of-list-bullets.adoc

```diff
@@ -0,0 +1,26 @@
+== Inline formatting.
+
+List bullets and list numbers can new be inline-formatted.
+
+Inline-formatting is the direct formatting of an object. It contrasts with formatting applied to an object because that object is a particular type of object.
+
+If all Table Headers are formatted a particular way, that is not inline formatting. When a particular Table Header is selected and has formatting specifically applied to it, that is inline formatting.
+
+Inline-formatting is also known as spot-formatting.
+
+If an end-user selects the entire content of a list item and applies inline formatting, such as a change of color or change of font-size, this spot-formatting is also applied to the list item’s associated bullet or number.
+
+Only the bullets or list numbers associated with the selection take on the inline-formatting applied to the selection.
+
+If the selection having inline formatting applied is one list item (that is, if the selection runs from an particular `<li>` tag to its associated `</li>` tag), the bullet or number associated with that single selected list item takes on the inline-formatting applied to the selection.
```

Comment:
Can this be simplified by breaking the info into separated sentences?

Reply:
> Can this be implied in the above paragraph instead?

It is more than implied, it is outright stated in the above paragraph. To wit:

*The above paragraph (with the outright statement emphasised)*  
> If an end-user selects the entire contents of a list item and
> applies inline formatting — such as a change of color or change of
> font-size — this spot-formatting is also applied **to the list
> item’s associated bullet or number**.

The following paragraph repeats this but also emphasises that **only** the associated list livery takes on the applied formatting.

There’s an old journalist’s maxim pertaining to getting a message across (that is, delivering information such that said information is actually taken in by the recipient):

1. Tell ’em.

2. Tell ’em you told ’em.

3. Tell ’em again.

It’s a truism because journalists, in common with technical writers, need to be aware their readers are, for the most part, only skimming their words.

So repetition is key to getting information across.

My two paragraphs here don’t quite follow the old maxim, because I’ve left out the second step, *tell ’em you told ’em*, but the repetition-with-variation is in step with the maxim, nonetheless.

Reply:
Re-framing this such that the parenthetical sub-clause is separate from the main sentence isn’t a bad idea.

But such re-framing is, to my mind an argument for turning this and the following paragraph into tables and adding displayed examples in the table. Something like as follows:
                                                                                                         
```
selection type        | underlying html selection | result of inline-formatting selection    
----------------------+---------------------------+--------------------------------------
1 list item           |                           |
more than 1 list item |                           |
entire list           |                           |
```